tests/functional/context_methods/test_cli_var_override.py

```diff
@@ -43,7 +43,7 @@ def test__override_vars_global(self, project):
 
 
 # This one switches to setting a var in 'test'
-class TestCLIVarOverridePorject:
+class TestCLIVarOverrideProject:
```

Comment:
This is a completely unrelated change to fix this typo. I'm curious what the culture is about bundling in tiny unrelated changes like this. Is it fine or is it expected to be broken out to a separate PR?